salt/_renderers/metalk8s_kubernetes.py

```diff
@@ -1,327 +1,108 @@
-'''
-A renderer for Kubernetes YAML manifests
+"""A renderer for Kubernetes YAML manifests.
 
 Given a Kubernetes YAML file (which may be a stream of objects, i.e. YAML
-snippets separated by `---` lines), this will render an SLS which calls Salt
-`*_present` states for every such object.
-
-To use it, add a shebang like `#!kubernetes` as the first line of your manifests
-SLS file. Optionally, you can use rendering pipelines (if templating is
-required), e.g. `#!jinja | kubernetes`.
-'''
-
-import base64
-import yaml
-
-import salt.utils.yaml
-from salt.utils.odict import OrderedDict
-
+snippets separated by `---` lines), this will render a sequence of states
+(represented as an OrderedDict), mapping every such object to an invocation
+of our custom `object_[present|absent]` state function.
+
+To use it, add a shebang like `#!kubernetes` as the first line of your
+manifests SLS file. Optionally, you can use rendering pipelines (if templating
+is required), e.g. `#!jinja | metalk8s_kubernetes`.
```

Comment:
Is it `kubernetes` (2 lines above) or `metalk8s_kubernetes`?

Reply:
`metalk8s_kubernetes` indeed I will update this